corehq/apps/sso/backends.py

```diff
@@ -84,6 +86,11 @@ def authenticate(self, request, username, idp_slug, is_handshake_successful):
         if not is_new_user and not web_user.is_active:
             web_user.is_active = True
             web_user.save()
+            log_user_change(by_domain=None, for_domain=None,
```

Comment:
"No longer adding changes to UserHistory because it is controversial."

Is the PR description still true?

Reply:
I see that since we have the two domain kwargs None, this would not be visible anywhere for projects though a record would still be added to `UserHistory` model.

I got a bit confused by that statement in the description. I'd suggest framing it as something like

`The change is not visible to projects since its a change to the web user which is not owned by any project and the change does not impact the project's membership for the user`

Reply:
Sorry for the confusion. I guess by saying "No longer adding changes to UserHistory because it is controversial." I mean "User History Report" 😂 
Updated the description as you suggested!

Reply:
well "controversial" is a strong word.

Thanks!

Reply:
Haha I don't know that is a strong word. 😂 I'll be more cautious about word selection next time!